config/services.php

```diff
@@ -256,4 +266,11 @@
             service(LayerAnalyser::class),
             service(Loader::class),
         ]);
+
+    $services
+        ->set(DebugUnassignedCommand::class)
+        ->args([
+                   service(UnassignedAnalyser::class),
+                   service(Loader::class),
+               ]);
```

Comment:
formatting is a bit off here

Reply:
Resolved in https://github.com/qossmic/deptrac/pull/572/commits/76d63941b73483bb7082e2f12cf1d95f4c75fd9d